modules/extensions/apps/portal/store/carbon.super/fs/gadget/user_id_publisher/js/action.js

```diff
@@ -0,0 +1,61 @@
+/*
+ ~ Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~      http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ */
+var jsonObj = [];
+var user;
+
+function loadUserList(dropdownId) {
+    var dropdownElementID = '#' + dropdownId;
+    user = getUrlVars()["user"];
+
+    $.ajax({
+        type: 'POST',
+        url: '../../bpmn-analytics-explorer/user_id_list',
+        success: function (data) {
+            if (!$.isEmptyObject(data)) {
+                var dataStr = JSON.parse(data);
+                for (var i = 0; i < dataStr.length; i++) {
+                    var opt = dataStr[i].assignUser;
+                    var el = document.createElement("option");
+                    el.textContent = opt;
+                    el.value = opt;
+                    $(dropdownElementID).append(el);
+                }
+                $('.selectpicker').selectpicker('val', user);
+                $(dropdownElementID).selectpicker("refresh");
+            }
+            else {
+                console.log('Empty User ID list.');
```

Comment:
Fixed in https://github.com/wso2/product-pc/commit/e931bb704e9065cf57f08f61619a45a4beecb5c7